Code/ForceField/UFF/testUFFForceField.cpp

```diff
@@ -1414,16 +1414,6 @@ void testUFFAllConstraints() {
   TEST_ASSERT(RDKit::feq(
       MolTransforms::getAngleDeg(mol->getConformer(), 1, 3, 6), 100.0, 0.5));
   delete field;
-  MolTransforms::setAngleDeg(mol->getConformer(), 1, 3, 6, 0.0);
```

Comment:
same comment as above: please keep the test, but use the new AngleConstraintContrib